File: test-claude-tdd/python/calculator.py
def add(a, b):
    """2つの数を加算する"""
    return a + b

def subtract(a, b):
    """2つの数を減算する"""
    return a - b

def multiply(a, b):
    """2つの数を乗算する"""
    return a * b

def divide(a, b):
    """2つの数を除算する"""
    if b == 0:
        raise ValueError("Division by zero is not allowed")
    return a / b

File: test-claude-tdd/go/calculator.go
package calculator

import "errors"

// Add returns the sum of two integers
func Add(a, b int) int {
	return a + b
}

// Subtract returns the difference of two integers
func Subtract(a, b int) int {
	return a - b
}

// Multiply returns the product of two integers
func Multiply(a, b int) int {
	return a * b
}

// Divide returns the quotient of two integers
func Divide(a, b int) (float64, error) {
	if b == 0 {
		return 0, errors.New("division by zero is not allowed")
	}
	return float64(a) / float64(b), nil
}